hoot-core-test/src/test/cpp/hoot/core/io/OsmApiDbTest.cpp

```diff
@@ -71,23 +84,36 @@ class OsmApiDbTest : public CppUnit::TestFixture
     }
   }
 
-  QUrl getOsmApiDbUrl()
+  //TODO: the sql output shouldn't be written out to null; simply checking the return value
+  //isn't enough to know whether it failed or not
```

Comment:
I'd really like to get rid of all those sql scripts in the code...not real fond of it being done that way... and have the data written by an actual writer, but that's a ton of work.